modules/social_features/social_profile/modules/social_profile_privacy/social_profile_privacy.module

```diff
@@ -119,6 +121,11 @@ function social_profile_privacy_admin_settings_form_submit($form, FormStateInter
   }
 
   $config->save();
+
+  // We need to update profile names.
+  /** @var \Drupal\social_profile_privacy\Service\SocialProfilePrivacyBatchHelper $batch_helper */
+  $batch_helper = \Drupal::service('social_profile_privacy.batch.helper');
+  $batch_helper->bulkUpdateProfileNames();
```

Comment:
We should be able to call a static function without calling it via services.

Reply:
@navneet0693 Check if it looks good to you now.